src/Steppenwolf/Shared/HeadComponent.razor

```diff
@@ -0,0 +1,18 @@
+﻿@inject HeadState head
+
+@foreach (var item in head.HeadItems)
+{
+    @((MarkupString)item)
```

Comment:
Do we need MarkupString?

Reply:
Yes, since this is injecting HTML in the head of the page
The headcomponent is a component that can inject CSS on the fly, thus optimizing pages. For example, the only page that needs the CSS for the multiple dropdowns is post create, and the CSS for ti will be injected only then.